portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/upgrades/OAuth2Upgrade.testcase

```diff
@@ -49,7 +47,7 @@ definition {
 		}
 	}
 
-	@description = "This is a use case for LRQA-61554."
+	@description = "This is a use case for LRQA-61554. Login to instance and check if OAuth2 works after upgrade from 7.2.0"
 	@priority = "5"
```

Comment:
I think that you can flag all tests with @uitest as well, as they are very similar to the first one

Reply:
Here is how to use @uitest label rule:
Make sure that at least one Poshi test is still testing basic UI flows. Mark these so we don't convert them as we refactor the rest of our tests
Use @uitest as an annotation for this

So I think mark one instance level test and one system level test as uitest is enough.